cmd/root.go

```diff
@@ -98,7 +98,7 @@ func Execute(cmd *cobra.Command, args []string) {
 	podmanRemove(types.PivotName)
 
 	// `podman mount` wants a running container, so let's make a dummy one
-	cid := utils.RunGetOutln("podman", "run", "-d", "--name",
+	cid := utils.RunGetOutln("podman", "run", "-d", "--net=none", "--name",
```

Comment:
Is ``--net`` valid? I've generally seen `--network`.

Reply:
It does seem valid!

Reply:
Yeah, though it's not in the man page for some reason. It's more common to me from the `docker` CLI.